src/Controls/src/Core/Compatibility/Handlers/iOS/FrameRenderer.cs

```diff
@@ -126,7 +126,10 @@ public virtual void SetupLayer()
 		public override void LayoutSubviews()
 		{
 			if (_previousSize != Bounds.Size)
+			{
 				SetNeedsDisplay();
+				this.UpdateBackgroundLayer();
```

Comment:
I'm getting a little bit lost on this PR.

We have a call [here](https://github.com/dotnet/maui/blob/21a4caf5e27f5debd0a0d4498eca556b7eeaef29/src/Core/src/Handlers/View/ViewHandler.iOS.cs#L10) that's supposed to update the background layer. But that calls `UpdateBackgroundLayerFrame` where as this calls `UpdateBackgroundLayer`. It seems like the `controls` layer generally needs a call into `UpdateBackgroundLayer` and that this isn't specific to just the `FrameRenderer`.  To fix this should we instead add a mapping addition at the controls layer to `Frame` that calls `UpgradeBackgroundLayer`?

Reply:
Is similar. This PR apply changes to update the Frame background layer with size changes. Because the Frame continue using a Renderer, the changes are applied directly to the Renderer code.

Reply:
Hmm, the `Frame` on iOS no longer uses this renderer. 

Frame on iOS uses this 

https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/Compatibility/Handlers/iOS/FrameRenderer.cs#L9

Which uses the handler-based renderer. 
This render is only used if the user adds `Frame` as a compatibility renderer.

Reply:
This code is using the handler based `VisualElementRenderer` though it's not a compatibility renderer so it's based on `IVEH`

Should this code go inside this class? 

https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/Compatibility/Handlers/iOS/VisualElementRenderer.cs#L10

The old renderer had it and it seems like it didn't get copied over?

<img width="646" alt="image" src="https://user-images.githubusercontent.com/5375137/203607567-1f85a00a-6610-4eda-9ce8-f59c7203e512.png">